tests/unit/jobserver/models/test_user.py

```diff
@@ -298,19 +298,32 @@ def test_user_ordering():
     user6 = UserFactory(fullname="Alice", username="CC")
 
     users = User.objects.all()
-    assert list(users) == [user2, user6, user4, user3, user5, user0, user1]
+    assert list(users) == [user2, user6, user4, user3, user5]
 
 
 def test_create_user():
     """Test the UserManager.create manager method."""
     username = "Ⅳan"  # Note initial unicode character.
     email = "tEsT@TeSt.tEST"  # Note uppercase characters.
     password = "hunter2"
-    user = User.objects.create(username=username, email=email, password=password)
+    fullname = "Test user"
+    user = User.objects.create(
+        username=username,
+        email=email,
+        password=password,
+        fullname=fullname,
+    )
 
     # Username unicode gets normalized.
     assert user.username == "IVan"
     # E-mail gets normalized by lower-casing.
     assert user.email == "test@test.test"
     # Password is not stored in plaintext.
     assert user.password != password
+
+
+def test_user_fullname_non_blank():
+    """Test the non-blank constraint on User"""
+    with pytest.raises(IntegrityError):
+        u = User(username="foo", email="foo@bar.com")
+        u.save()
```

Comment:
I think this effectively tests the expected behaviour, but we may as well make the use of `fullname` as explicit as possible in the docstring and `init` call.

```suggestion
    """Test the non-blank constraint on User.fullname."""
    with pytest.raises(IntegrityError):
        u = User(username="foo", email="foo@bar.com", fullname="")
        u.save()
```

Reply:
Good idea, I'll do that